enterprise_gateway/itests/kernels/authorization_complete/kernel.json

```diff
@@ -0,0 +1,20 @@
+{
+  "display_name": "Authorization Testing",
+  "language": "python",
+  "process_proxy": {
+    "class_name": "enterprise_gateway.services.processproxies.processproxy.LocalProcessProxy",
+    "config": {
+      "authorized_users": "authorized_user_1,authorized_user_2",
+      "unauthorized_users": "unauthorized_user_1,unauthorized_user_2"
```

Comment:
It is important to ensure that denial takes place if an unauthorized user is **also** in the list of authorized users.  We must ensure denials take precedence over allows.  Please add `unauthorized_user_1` to the `authorized_users` list.